tests/e2e/features/artifacts/transferPage/atrifacts-transfers.feature

```diff
@@ -147,3 +147,34 @@ Feature: Artifacts - UI
     Then Element with "text" "ZZ" should be "visible"
     Then Element with "text" "0x1ab721...184" should be "visible"
     Then Element with "partial src" "zz.png" should be "visible"
+
+  @id1537 @id1404
+  Scenario: Check artifacts on Transaction submitted pop up/Transaction completed pop up (Transfer)
+    Given I am on the Main page
+    When I go to page "/transaction/zksync/era/send?network=era-goerli&address=0x2CF4F78917A16C9584AeB5d4c5bD2713d724C75d"
+    When I choose "ETH" as token and insert "0.0000000001" as amount
+    When I "confirm" transaction after clicking "Send to zkSync Era Testnet" button
```

Comment:
Do we need to add these preparation step above in description? https://allure.matterlabs.dev/project/17/test-cases/1537?treeId=30

Reply:
We already have "Pass Transfer frow (TC1321) till the "Transaction submitted" pop up (complete transfer transaction)" in "Description" step